bundles/com.espressif.idf.ui/src/com/espressif/idf/ui/tools/ToolsInstallationHandler.java

```diff
@@ -557,9 +560,24 @@ public Boolean call() throws Exception
 			configureToolChain();
 			configEnv();
 			copyOpenOcdRules();
+			updateEspressifPrefPageOpenocdPath();
 			return Boolean.TRUE;
 		}
 
+		private void updateEspressifPrefPageOpenocdPath()
+		{
+			IEclipsePreferences newNode = DefaultScope.INSTANCE.getNode("com.espressif.idf.debug.gdbjtag.openocd"); //$NON-NLS-1$
+			newNode.put("install.folder", IDFUtil.getOpenOCDLocation()); //$NON-NLS-1$
+			try
+			{
+				newNode.flush();
+			}
+			catch (BackingStoreException e)
+			{
+				Logger.log(e);
+			}
+		}
+		
```

Comment:
As this code is the same for Install Tools and Tools Installation Wizard, could we keep this in IDFUtil and call from both classes?

Reply:
Hi @kolipakakondal I have incorporated changes as you mentioned I have seen other methods that can be moved out to a common class as well which I will do in other ticket.